OctoKit/Milestone.swift

```diff
@@ -67,3 +68,397 @@ open class Milestone: Codable {
         case dueOn = "due_on"
     }
 }
+
+// MARK: Request
+
+public extension Octokit {
+    /**
+     Create a milestone
+     - parameter owner: The user or organization that owns the repositories.
+     - parameter repo: The name of the repository.
+     - parameter title: The title of the new milestone.
+     - parameter state: Filter pulls by their state.
```

Comment:
This seems to be incorrect. Creating a milestone should have no filter?

Reply:
The docs are wrong, but create milestone endpoint accepts `state` value
https://docs.github.com/en/rest/issues/milestones?apiVersion=2022-11-28#create-a-milestone